Tests/test_MafIO_index.py

```diff
@@ -302,19 +365,24 @@ def test_correct_retrieval_1(self):
                     3014842, 1371, 7842, 171548, 16169512, 16389874, 15871306, 6404, 184317, 14750994,
                     3018161, 16390178, 15871611, 16169818,
                     3018230, 15871676, 16390243,
-                    3018359, 16390338, 15871771, 184712, 16169976, 3018482})
+                    3018359, 16390338, 15871771, 184712, 16169976, 3018482
+                }
+            )
+            # Turn black code style on
+            # fmt: on
```

Comment:
You could try adding comment lines as break points instead, but black would still want to use one line per value.

So what you've done is probably best (turning off black here).